quickstart.md

```diff
@@ -380,618 +380,173 @@ $ ssh -i ccv0-ssh root@$(kubectl get service ccv0-ssh -o jsonpath="{.spec.cluste
-
-```
-measurement=$(sev-snp-measure --mode=sev --output-format=base64 \
-  --ovmf="${ovmf_path}" \
-  --kernel="${kernel_path}" \
-  --initrd="${initrd_path}" \
-  --append="${append}" \
-)
-```
-
-If the container image is not already present, pull it:
-
-```
-encrypted_image_url="ghcr.io/fitzthum/encrypted-image-tests:unencrypted"
-docker pull "${encrypted_image_url}"
-```
-
-Retrieve the encryption key from docker image label:
-
-```
-enc_key=$(docker inspect ${encrypted_image_url} \
-  | jq -r '.[0].Config.Labels.enc_key')
-```
-
-Add the key, keyset and policy with measurement to the `simple-kbs` database:
-
-```
-mysql -u${KBS_DB_USER} -p${KBS_DB_PW} -h ${KBS_DB_HOST} -D ${KBS_DB} <<EOF
-  REPLACE INTO secrets VALUES (10, 'key_id1', '${enc_key}', 10);
-  REPLACE INTO keysets VALUES (10, 'KEYSET-1', '["key_id1"]', 10);
-  REPLACE INTO policy VALUES (10, '["${measurement}"]', '[]', 0, 0, '[]', now(), NULL, 1);
-EOF
-```
-
-Using the same service yaml from the section on [Launch the Pod and Verify SEV Encryption](#launch-the-pod-and-verify-sev-encryption), launch the service:
-
-```
-kubectl apply -f encrypted-image-tests.yaml
-```
-
-Check for pod errors:
-
-```
-pod_name=$(kubectl get pod -o wide | grep encrypted-image-tests | awk '{print $1;}')
-kubectl describe pod ${pod_name}
-```
-
-The pod will error out on the key retrieval request to the `simple-kbs` because the policy verification failed due to a mismatch in the SEV guest firmware measurement. This is the error message that should display:
-
-```
-Policy validation failed: fw digest not valid
-```
-
-The `PLACEHOLDER` value that was set for the kernel append line when the SEV guest firmware measurement was calculated does not match what was measured by the kata shim. The kernel append line parameters can be retrieved from the qemu command line using the following scripting commands, as long as kubernetes is still trying to launch the pod:
-
-```
-duration=$((SECONDS+30))
-set append
-
-while [ $SECONDS -lt $duration ]; do
-  qemu_process=$(ps aux | grep qemu | grep append || true)
-  if [ -n "${qemu_process}" ]; then
-    append=$(echo ${qemu_process} \
-      | sed "s|.*-append \(.*$\)|\1|g" \
-      | sed "s| -.*$||")
-    break
-  fi
-  sleep 1
-done
-
-echo "${append}"
+```json
+{
+    "default": [{"type": "reject"}], 
+    "transports": {
+        "docker": {
+            "[REGISTRY_URL]": [
+                {
+                    "type": "sigstoreSigned",
+                    "keyPath": "kbs:///default/cosign-key/1"
+                }
+            ]
+        }
+    }
+}
 ```
 
-The above check will only work if the `encrypted-image-tests` guest launch is the only consuming qemu process running.
+Be sure to replace `[REGISTRY_URL]` with the desired registry URL of the encrypted image.
 
-Now, recalculate the SEV guest firmware measurement and store the `simple-kbs` policy in the database:
+Register the image pulling validation policy file to KBS storage:
```

Comment:
We also need to register the verification key to KBS storage, too

Reply:
Yes,  guide of this is at line 480 :-)